concept/ConceptManager.java

```diff
@@ -52,22 +52,22 @@ public ConceptManager(RPCTransaction rpcTransaction) {
 
     @CheckReturnValue
     public ThingType getRootThingType() {
-        return getType(GraqlToken.Type.THING.toString()).asThingType();
+        return getThingType(GraqlToken.Type.THING.toString()).asThingType();
```

Comment:
can be simplified to 
```suggestion
        return getThingType(GraqlToken.Type.THING.toString());
```
There are more examples further down the file.

Reply:
Fixed in https://github.com/graknlabs/client-java/pull/211/commits/ace3adaff902dab2bca5d1d1961d29e164c4ddf8.